lib/pages/saved_places.dart

```diff
@@ -0,0 +1,375 @@
+                log('bttn');
+              },
+              child: Row(
+                children: <Widget>[
+                  Container(
+                      margin: const EdgeInsets.only(right: 5),
+                      child: Icon(
+                          _typeToIconData(savedPlace.type) ?? Icons.place)),
+                  Expanded(
+                    child: Text(
+                      savedPlace.description,
+                      style: theme.textTheme.body2,
+                      maxLines: 1,
+                    ),
+                  ),
+                  PopupMenuButton<int>(
+                    itemBuilder: (BuildContext context) => [
+                      PopupMenuItem(
+                        value: 1,
+                        child: Text(localization.savedSectionSetIcon()),
+                      ),
+                      PopupMenuItem(
+                        value: 2,
+                        child: Text(localization.savedSectionSetCoordinate()),
+                      ),
+                      PopupMenuItem(
+                        value: 3,
+                        child: Text(localization.savedSectionRemove()),
+                      ),
+                    ],
+                    onSelected: (int index) async {
+                      if (index == 1) {
+                        _changeIcon(savedPlace);
+                      } else if (index == 2) {
+                        _changeCoordinate(savedPlace);
+                      } else if (index == 3) {
+                        setState(() {
+                          savedLocationsBloc.inRemoveLocation.add(savedPlace);
+                        });
+                      }
+                    },
+                  ),
+                ],
+              ),
+            ),
+          );
+        },
+        childCount: data.length,
+      ),
+    );
+    final List<Widget> slivers = <Widget>[];
+    slivers.add(saveds);
+
+    return CustomScrollView(slivers: slivers);
+  }
+
+  Widget _addPlaceBttn() {
+    final ThemeData theme = Theme.of(context);
+    final TrufiLocalization localization =
+        TrufiLocalizations.of(context).localization;
+
+    return RaisedButton(
+      onPressed: () async {
+        final LatLng mapLocation = await Navigator.of(context).push(
+          MaterialPageRoute<LatLng>(
+            builder: (BuildContext context) => ChooseLocationPage(
+              initialPosition: _center,
+            ),
+          ),
+        );
+        if (mapLocation != null) {
+          _addPlace(mapLocation);
+        }
+      },
+      color: theme.primaryColor,
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+        children: <Widget>[
+          Text(
+            localization.savedSectionAddPlace(),
+            style: theme.primaryTextTheme.body2,
+          ),
+          Icon(
+            Icons.add_to_home_screen,
+            color: theme.backgroundColor,
+          ),
+        ],
+      ),
+    );
+  }
```

Comment:
Haha, the document said "something else", you've taken that quite literal :D

I think a Floating Action Button with the `add` icon would work better here though.
https://api.flutter.dev/flutter/material/FloatingActionButton-class.html